pos-print/src/main/java/com/clouway/pos/print/core/ReceiptPrintWorker.kt

```diff
@@ -0,0 +1,20 @@
+package com.clouway.pos.print.core
+
+/**
+ * Provides the methods to print a collection of receipts
+ *
+ * @author Tsvetozar Bonev (tsvetozar.bonev@clouway.com)
+ */
+interface ReceiptPrintWorker {
```

Comment:
Not sure that you need this interface at all.

Reply:
Agreed, but if I remove this interface from the background thread and leave it as a plain AbstractExecutionThreadSerivce, how would I go about binding it's injections with Guice?

Reply:
Do you need such binding at all?

Reply:
btw, you can bind and classes too.

Reply:
I do not, my mistake, will remove the interface.